{{cookiecutter.repo_name}}/src/pages/account/change-password.jsx

```diff
@@ -0,0 +1,126 @@
+                }}{% endraw %}
+                className={classes.form}
+                validateOnChange
+                validationSchema={ChangePassSchema}
+                onSubmit={(values, actions) => {
+                    changePass(values)
+                        .then(response => {
+                            enqueueSnackbar('Successfully changed password!', {
+                                variant: 'success',
+                            });
+                            return response;
+                        })
+                        .then(response => {
+                            actions.setSubmitting(false);
+                            return response;
+                        })
+                        .catch(error => {
+                            actions.setSubmitting(false);
+                            if (error.non_field_errors) {
+                                enqueueSnackbar(error.non_field_errors, {
+                                    variant: 'error',
+                                });
+                            } else {
+                                actions.setErrors(error);
+                            }
+                        });
+                }}
+            >
+                <Form>
+                    <Grid container spacing={2} className={classes.grid}>
+                        <Grid item xs={12} sm={12}>
+                            <Field
+                                fullWidth
+                                component={TextField}
+                                name="current_password"
+                                autoComplete="current-password"
+                                label="Current Password"
+                                type="password"
+                            />
+                        </Grid>
+
+                        <Grid item xs={12} sm={6}>
+                            <Field
+                                fullWidth
+                                name="new_password"
+                                component={TextField}
+                                autoComplete="new-password"
+                                type="password"
+                                label="New password"
+                            />
+                        </Grid>
+                        <Grid item xs={12} sm={6}>
+                            <Field
+                                fullWidth
+                                name="re_new_password"
+                                component={TextField}
+                                type="password"
+                                autoComplete="new-password"
+                                label="Repeat New password"
+                            />
+                        </Grid>
+                    </Grid>
+                    <Button
+                        fullWidth
+                        variant="outlined"
+                        type="submit"
+                        className={classes.bottomSpace}
```

Comment:
Same for any class that just uses bottomspace. I dunno. It actually means more lines of code but wondering about what's the best thing to model here for future devs using it. Like do we want people to fall back to Box or to use makestyles